app/authenticators/drf-token-authenticator.js

```diff
@@ -16,34 +20,6 @@ export default Base.extend({
     });
   },
 
-  authenticate(username, password) {
```

Comment:
With the `authenticate()` method removed, is there still a simple way to authenticate without relying on bespin-api setting a cookie? I'm thinking mostly for development purposes when the applications may be running under different debug processes on different hosts/ports and wouldn't share cookies. 

Is there any harm in leaving it (and the login page) in, wrapped in a check to see if the current environment is `development`?